ec2

```diff
@@ -15,7 +14,8 @@ function get_cache() {
 }
 
 cache=$(get_cache)
-export chosen=$(echo "$cache" | fzf -q "$DEFAULT_FZF_QUERY")
+chosen=$(echo "$cache" | fzf -q "$DEFAULT_FZF_QUERY")
+export chosen
```

Comment:
why??

Reply:
the return value of the shorten notation `export x=y` is that of the export, which is always good. therefore errors in `fzf` (such as due to ctrl-c) are not caught by `-e`

Reply:
TIL...